### PR TITLE
Login via Payload

### LINKED ISSUE #335: Authenticate with Payload
We need some way to authenticate with payload whilst **also** remaining logged in for pages we haven't moved over yet

### PR DESCRIPTION
**Pull request by rugulous**
Fixes #335 

(Also needs https://github.com/BritishYouthBandAssociation/Library/pull/161)

### CODE REVIEW

File: views/login.hbs [hunk 1]
```diff
@@ -24,4 +21,63 @@
...
+			credentials: 'include',
+			body: JSON.stringify({
+				email: email.value,
+				password: password.value
+			})
+		});
+
+		let json = await response.json();
+
+		if(response.status / 100 != 2){
```
Comment (matt-fidd) [inline views/login.hbs:57]: this is just the same as `=200` right?

File: views/login.hbs [hunk 2]
```diff
@@ -24,4 +21,63 @@
...
+				"Content-Type": "application/json"
+			},
+			body: JSON.stringify({
+				email: email.value,
+				password: password.value,
+				payload: json
+			})
+		});
+
+		if(response.status / 100 != 2){
```
Comment (matt-fidd) [inline views/login.hbs:76]: same here

File: views/login.hbs (hunk 1 above)
Comment (rugulous) [reply views/login.hbs:57]: To be fair yeah lmao. I originally had `== 4` so it would catch any errors but I suppose the divisions aren't needed now!

**Comment by rugulous** [comment]
@matt-fidd should be good now :)

**Review by matt-fidd — APPROVED** [review]

**Pull request merged**